Lib/ufo2ft/makeotfParts.py

```diff
@@ -156,6 +154,19 @@ def setupAnchorPairs(self):
 
         self.mkmkAnchorPairs = self.anchorPairs
 
+    def _findLayoutFeatures(self):
+        """Returns what OpenType layout feature tags are present in the UFO."""
+        if self.font.path is None:
+            return set()
+        feapath = os.path.join(self.font.path, "features.fea")
+        if not os.path.exists(feapath):
+            return set()
+        glyphMap = self.outline.getReverseGlyphMap()
+        parser = feaLib.parser.Parser(feapath, glyphMap=glyphMap)
```

Comment:
I see, that makes sense.

(I was looking at the [CoffeeScript compiler](https://github.com/cha0s/coffeescript/blob/60d0a446cd46de0cd4ae790e9d8bd4382b8fc9c8/src/coffee-script.coffee#L70) the other day where one can call compile() or call tokenizer, parser, compiler as separate steps. This way you can look at the ast [or even edit it?] before doing the compilation step.)